content/Entities/Tools/Axe/prefab.axe.hjson

```diff
@@ -39,6 +40,7 @@ melee:
 	damage_type: axe
 	category: bladed
 	hit_mask: world, destructible
+	auto_attack: true
```

Comment:
May want to use enum flags for this instead (bool isn't blittable in C#, so it often screws up memory layout of structs - I'll probably make an analyzer for it one day)